CONTRIBUTING.md

```diff
@@ -2,29 +2,31 @@
 
 ## Feedback
 
-The report of issues and the proposal of new features is very welcome.
-Please use the GitHub issue page for this.
+Issue reports and feature proposals are very welcome.
+Please use the [GitHub issue page](https://github.com/glotzerlab/signac-flow/issues/) for this.
 
 ## Contributing code
 
-Code contributions to the signac-flow open-source project are welcomed via pull requests on GitHub.
-Prior any work you should contact the signac-flow developers to ensure that the planned development meshes well with the directions and standards of the project.
+Code contributions to the signac open-source project are welcomed via pull requests on GitHub.
```

Comment:
Do we want this to say `signac-flow` or not? (is it the package or the framework?)

Reply:
I intentionally just left it general there..